src/ui/Header.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import SettingsCard from './SettingsButton';
+import {
+  Flex,
+  Link,
+  HStack,
+  Button as ChakraButton,
+  Text,
+} from '@chakra-ui/react';
+import { Icon, IconNames } from '@nypl/design-system-react-components';
+import { WebpubManifest, ReaderState, Navigator } from '../types';
+
+export type HeaderProps = {
+  logo?: any;
+  manifest: WebpubManifest; // for TOC?
```

Comment:
I'd say get rid of this for now.  It's currently not being used at all (L21) and we can add it back in when we know what we need it for

Reply:
That makes sense. Updated.